modules/aws-api-gateway/variables.tf

```diff
@@ -0,0 +1,15 @@
+variable "environment" {
+  description = "The environment name (e.g., dev, staging, prod)"
+  type        = string
+}
+
+variable "name" {
+  description = "The name prefix for resources created by this module"
+  type        = string
+}
+
+variable "tags" {
+  description = "A map of tags to assign to the API Gateway"
+  type        = map(string)
+  default     = {}
+} 
```

Comment:
I've been defaulting this to the no cost center:

https://github.com/Sage-Bionetworks-Workflows/eks-stack/blob/db9e56b88118c168567163eae19523a70653afa2/modules/s3-bucket/variables.tf#L6-L12